manifest.yaml

```diff
@@ -222,4 +222,8 @@ arguments:
       type: string
     default: "0.14.11"
     description: The version of terraform that is used.
+  disableGrpcGeneration:
+    description: Disable gRPC generation if you are using protoAPI generation
```

Comment:
What's protapi? To an open source user this is confusing, I'd just say diable grpc client generation for the service

Reply:
@pavelsmejkal Could you please explain better?